### ISSUE TITLE
[Feature]: Divided Stairs "Shape" blockstate.

### ISSUE THREAD

**Issue by Pumciusz** [XFactHD/FramedBlocks#338]
### Describe the feature

 Hello, while playing on 9.1.5 version for Minecraft 1.20.1 (haven't updated coz too lazy to solve modlist issues) I noticed that you can change the "Shape" blockstate of regular framed stairs, either by hand with a debug stick, or just by placing them and rotating via the Framed Wrench, but you can't with Divided Stairs and Vertical Divided Stairs. I also checked on a new mod instance with 9.1.6 version.
 Is it possible for the Divided Stairs to behave more like regular stairs, or there would have to be a new block that adds that function?
Basically what I'm looking for is a Framed Panel and Framed Half Stair in a sinlge block.

Thanks in advance, Pictures for context.
![shape](https://github.com/XFactHD/FramedBlocks/assets/149726309/92190ad5-5034-4a7d-aea7-5b14b21fcf4e)
![block](https://github.com/XFactHD/FramedBlocks/assets/149726309/e98f83d8-dd75-4b9d-adf8-0b25f6397191)

**Comment by XFactHD** [comment]
I have been pondering about adding behaviour like this multiple times but every time it comes down to unfortunately not being feasible for technical reasons.
